src/system-plugins/rov-beacon/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
This is the service plugin that runs on the ROV to collect the data that is relevant to the beacon message and to then broadcast it.